tethys_gizmos/static/tethys_gizmos/js/select_input.js

```diff
@@ -25,7 +25,11 @@ var TETHYS_SELECT_INPUT = (function() {
  	var initSelectInput;
 
  	initSelectInput = function(tag) {
-        $(tag).select2();
+ 	    $(tag).each(function(){
```

Comment:
This is to make sure the functionality works correctly when being called from AJAX. Since I'm doing it here I could remove the .each from line 45ish. Thoughts?